JeroenArsenal/JNA/fn_arsenal_loadInventory.sqf

```diff
@@ -350,29 +308,17 @@ _containers = [_uniform,_vest,_backpack];
 
 		call {
 			if ([_itemCounts select _index, _item] call jn_fnc_arsenal_itemCount == -1) exitWith {
-				call ([
-					{player forceAddUniform _uniform;},//todo remove
```

Comment:
Theese lines were repeated multiple times, they are regrouped into line 296 with the private that can be just called without repeating.